src/nplinker/pairedomics/downloader.py

```diff
@@ -315,34 +318,33 @@ def _load_gnps_data(self, gnps_task_id) -> zipfile.ZipFile:
 
         # Try read from cache
         if os.path.exists(self.metabolomics_zip):
-            logger.info('Found existing metabolomics_zip at {}'.format(
-                self.metabolomics_zip))
+            logger.info('Found existing metabolomics_zip at %s',
+                        self.metabolomics_zip)
             try:
-                mbzip = zipfile.ZipFile(self.metabolomics_zip)
-                return mbzip
-            except zipfile.BadZipFile as bzf:
+                with zipfile.ZipFile(self.metabolomics_zip) as mbzip:
+                    return mbzip
```

Comment:
It's not right to use `with` when you want to return a opened file, since `with` will automatically close the file. The original code is right.

Reply:
The original code is there now, but I disabled the pylint warning about consider-using-with